shared-module/ads1x9x/ADS1x9x.c

```diff
@@ -324,3 +325,38 @@ size_t common_hal_ads1x9x_ADS1x9x_read(ads1x9x_ADS1x9x_obj_t *self, mp_buffer_in
     memcpy(ptr, g_ads_buffer, buf_size);
     return buf_size;
 }
+
+void set_emg_filter(ads1x9x_ADS1x9x_obj_t *self, mp_obj_t coeffs_list, mp_obj_t low) 
```

Comment:
Should those functions leave in emg_iir.c ?

Reply:
moved